src/plugin/ogc/ui/geoserverimport.js

```diff
@@ -75,3 +77,17 @@ plugin.ogc.ui.GeoserverImportCtrl.prototype.getDataProvider = function() {
 plugin.ogc.ui.GeoserverImportCtrl.prototype.getUrl = function() {
   return this.dp ? /** @type {plugin.ogc.GeoServer} */ (this.dp).getOriginalWmsUrl() : '';
 };
+
+
+/**
+ * @inheritDoc
+ * @export
+ */
+plugin.ogc.ui.GeoserverImportCtrl.prototype.validateUrl = function() {
+  if (/\/web\/?$/.test(this.scope['config']['url'])) {
+    this.scope['customUrlMessage'] = 'GeoServer URLs ending with \"/web\" are typically for the administration ' +
+    'interface. Consider replacing \"/web\" with \"/wfs,\" \"/wms,\" or \"/wmts\" for the OGC service APIs.';
```

Comment:
OpenSphere only supports the `/owc` endpoint for the GeoServer import path so we shouldn't advertise these here. You can use them if you are adding the server via Add Server -> OGC Server, but here they will still produce an error.